sound/soc/sof/intel/hda-dai.c

```diff
@@ -57,8 +57,8 @@ static bool hda_check_fes(struct snd_soc_pcm_runtime *rtd,
 }
 
 static struct hdac_ext_stream *
-	hda_link_stream_assign(struct hdac_bus *bus,
-			       struct snd_pcm_substream *substream)
+hda_link_dma_stream_assign(struct hdac_bus *bus,
```

Comment:
If the Intel documentation internally refers to 'HDA DMA link' then would it make sense to use 'hda_dma_link` as prefix for the same thing?

Reply:
it's a typo, the nomenclature is 'hda host dma' and 'hda link dma'. will update the comment.